UnitsNet.Tests/GeneratedCode/TestsBase/VolumeTestsBase.g.cs

```diff
@@ -1971,1076 +1971,358 @@ public void TryParse()
+        [InlineData("en-US", "nl", VolumeUnit.Nanoliter)]
+        [InlineData("en-US", "bbl", VolumeUnit.OilBarrel)]
+        [InlineData("en-US", "bl (U.S.)", VolumeUnit.UsBeerBarrel)]
+        [InlineData("en-US", "gal (U.S.)", VolumeUnit.UsGallon)]
+        [InlineData("en-US", "oz (U.S.)", VolumeUnit.UsOunce)]
+        [InlineData("en-US", "pt (U.S.)", VolumeUnit.UsPint)]
+        [InlineData("en-US", "qt (U.S.)", VolumeUnit.UsQuart)]
+        [InlineData("ru-RU", "сл", VolumeUnit.Centiliter)]
+        [InlineData("ru-RU", "см³", VolumeUnit.CubicCentimeter)]
+        [InlineData("ru-RU", "дм³", VolumeUnit.CubicDecimeter)]
+        [InlineData("ru-RU", "фут³", VolumeUnit.CubicFoot)]
+        [InlineData("ru-RU", "дюйм³", VolumeUnit.CubicInch)]
+        [InlineData("ru-RU", "м³", VolumeUnit.CubicMeter)]
+        [InlineData("ru-RU", "мкм³", VolumeUnit.CubicMicrometer)]
+        [InlineData("ru-RU", "миля³", VolumeUnit.CubicMile)]
+        [InlineData("ru-RU", "мм³", VolumeUnit.CubicMillimeter)]
+        [InlineData("ru-RU", "ярд³", VolumeUnit.CubicYard)]
+        [InlineData("ru-RU", "дал", VolumeUnit.Decaliter)]
+        [InlineData("ru-RU", "даАмериканский галлон", VolumeUnit.DecausGallon)]
+        [InlineData("ru-RU", "дл", VolumeUnit.Deciliter)]
+        [InlineData("ru-RU", "дАмериканский галлон", VolumeUnit.DeciusGallon)]
+        [InlineData("ru-RU", "гфут³", VolumeUnit.HectocubicFoot)]
+        [InlineData("ru-RU", "гл", VolumeUnit.Hectoliter)]
+        [InlineData("ru-RU", "гАмериканский галлон", VolumeUnit.HectousGallon)]
+        [InlineData("ru-RU", "Английский галлон", VolumeUnit.ImperialGallon)]
+        [InlineData("ru-RU", "Английская унция", VolumeUnit.ImperialOunce)]
+        [InlineData("ru-RU", "кфут³", VolumeUnit.KilocubicFoot)]
+        [InlineData("ru-RU", "кАнглийский галлон", VolumeUnit.KiloimperialGallon)]
+        [InlineData("ru-RU", "кл", VolumeUnit.Kiloliter)]
+        [InlineData("ru-RU", "кАмериканский галлон", VolumeUnit.KilousGallon)]
+        [InlineData("ru-RU", "л", VolumeUnit.Liter)]
+        [InlineData("ru-RU", "Мфут³", VolumeUnit.MegacubicFoot)]
+        [InlineData("ru-RU", "МАнглийский галлон", VolumeUnit.MegaimperialGallon)]
+        [InlineData("ru-RU", "Мл", VolumeUnit.Megaliter)]
+        [InlineData("ru-RU", "МАмериканский галлон", VolumeUnit.MegausGallon)]
+        [InlineData("ru-RU", "мкл", VolumeUnit.Microliter)]
+        [InlineData("ru-RU", "мл", VolumeUnit.Milliliter)]
+        [InlineData("ru-RU", "нл", VolumeUnit.Nanoliter)]
+        [InlineData("ru-RU", "Американский галлон", VolumeUnit.UsGallon)]
+        [InlineData("ru-RU", "Американская унция", VolumeUnit.UsOunce)]
+        [InlineData("fr-CA", "pmp", VolumeUnit.BoardFoot)]
+        [InlineData("fr-CA", "pied-planche", VolumeUnit.BoardFoot)]
+        [InlineData("fr-CA", "pied de planche", VolumeUnit.BoardFoot)]
+        public void ParseUnitWithCulture(string culture, string abbreviation, VolumeUnit expectedUnit)
```

Comment:
If you want to make it really explicit- we could aslo include the non-localized mappings for `(culture, englishAbbreviation, unit)`:
```  csharp
        [InlineData("fr-CA", "ac-ft", VolumeUnit.AcreFoot)]
        [InlineData("fr-CA", "acre-foot", VolumeUnit.AcreFoot)]
        [InlineData("fr-CA", "acre-feet", VolumeUnit.AcreFoot)]
        ....
```